From 8d22da4fdcbabcc530ae902614ca51d505c5851b Mon Sep 17 00:00:00 2001
From: Michal Suchanek <msuchanek@suse.de>
Date: Wed, 12 Sep 2018 19:06:23 +0200
Subject: [PATCH 16/24] Define early_radix_enabled()

References: bsc#1094244
Patch-mainline: no, not needed

We do not support radix so radix is never enabled.

Signed-off-by: Michal Suchanek <msuchanek@suse.de>
---
 arch/powerpc/include/asm/mmu.h | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/powerpc/include/asm/mmu.h b/arch/powerpc/include/asm/mmu.h
index 8f6c6fa7cce6..431799d62af7 100644
--- a/arch/powerpc/include/asm/mmu.h
+++ b/arch/powerpc/include/asm/mmu.h
@@ -150,6 +150,11 @@ static inline void assert_pte_locked(struct mm_struct *mm, unsigned long addr)
 }
 #endif /* !CONFIG_DEBUG_VM */
 
+static inline bool early_radix_enabled(void)
+{
+	return false;
+}
+
 #endif /* !__ASSEMBLY__ */
 
 /* The kernel use the constants below to index in the page sizes array.
-- 
2.13.7

